### DISCUSSION TITLE
Asking for your views on layout

### DISCUSSION THREAD

**Discussion by MichaelCade** [MichaelCade/90DaysOfDevOps#189] [Announcements]
Hey Everyone, 

There has been an incredible amount of contribution from the community on this repository and I have said quite a lot on podcasts, live streams and in general that I am missing a lot to do with Security which has paved its way to the start of 2023 which is fast approaching. 

My ask is how do we display the next phase of the 90Days? 

We currently have our readme.md which contains links to the individual day's pages and it was great to follow along or now see each section. The content here is still massively valuable and relevant to the community for learning so I don't want this sinking to the bottom of the page. 

First idea: Create a new readme which provides us with an option to head to the 2022 completed section where we point to the now existing readme.md, and then we create a new readme for 2023 so we then have a clean readme when people come to the repository. 

Second idea: we are limited in markdown (At least I think) in that we can use tables but it becomes quite messy. but it is an option. 

Another thing to think about here is the Days folder, we currently use this to store our 2022 days md files and then the subsection folders where code bases and examples are stored. How would be best to do this for the 2023 version? Should we create a new folder in the root called 2023_Days? and then over time possibly change the existing Days folder to 2022? This would be a lot of work now with a lot of translations taking place also. 

If anyone has thoughts or examples of others, I am open as I am starting to get things going again for 2023... I must be mad!

**Comment by MichaelCade** [comment]
Here is something that looks to be near the first idea - https://github.com/MichaelCade/90DaysOfDevOps/tree/90DaysOfDevSecOps

**Comment by Lexpeartha** [comment]
Not sure if this is outlandish idea, but maybe taking the project onto a website would be useful? Content would still be fetched from markdown, but it would be way more interactive.

Also I think idea of different "chapters" could be cool, maybe have different folders for 2022, 2023 and etc.
and maybe each year could have all files it needs under it:

```sh
/2023
- /en
- /ja
# and etc
```

**Reply by MichaelCade** [reply]
I think the folder for years is needed. Although quite a bit of work to make all of the changes. 

Interested in hearing more regarding the website idea and what you think would enable more interaction. I do already have the domain 90DaysOfDevOps.com which currently points to the repository.

**Reply by Lexpeartha** [reply]
Maybe something like [tech-interview-handbook](https://www.techinterviewhandbook.org), they have website for reading handbook, which makes it very easy to track progress, go around table of content that's on the side. And maybe there could be auth with an account so you can track your progress?

Just throwing ideas out there, I have nothing specific in mind

**Reply by MichaelCade** [reply]
@Lexpeartha have you seen the progress we have been making around the [website](http://www.90daysofdevops.com/#/) we are going to create a GH-Pages branch to fix some of the navigation issues but so far this seems nicer than the GitHub repository navigation. Let us know your thoughts.

**Comment by eddiejaoude** [comment]
What about using something like NextJS or Astro?
